concert/base.py

```diff
@@ -860,3 +861,61 @@ def unlock(self):
         """Unlock all the parameters for writing."""
         for param in self:
             param.unlock()
+
+
+_INSTANCE_CACHE = False
+
+
+def _memoize_instance_id(func):
```

Comment:
Of course not, I was so happy I found the `memoize` in helpers that I wanted to use it but only later I realized I need a new one but to rethink the decorator usage somehow wasn't scheduled in my head :smile: 
Anyway, now it's there decoratorless.